web/modules/custom/contact_information/contact_information.info.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
 config_devel:
   - core.entity_form_display.rdf_entity.contact_information.default
   - core.entity_form_display.rdf_entity.contact_information.inline
+  - core.entity_view_display.rdf_entity.contact_information.about
```

Comment:
2 notes on current display: 
- In wireframe, grey areas are 1 area (owner and contact)
- when multiple items in lists, markup is not showing a list, it's not the best semantically

I'm not going to stop the pull for these 2, but please check whether they are a big problems for next stages of approval.

![solution_about](https://user-images.githubusercontent.com/1923476/28075181-04250716-665b-11e7-9d9b-5667a9661b7d.png)